pydoctor/test/epydoc/test_restructuredtext.py

```diff
@@ -1,8 +1,11 @@
+import pprint
 from typing import List
-
-from pydoctor.epydoc.markup import ParseError, flatten
+from unittest import skip
```

Comment:
The imported `skip` isn't used. But while pytest has some support for the built-in `unittest` framework, I think it's better to use [pytest's native markers](https://docs.pytest.org/en/stable/skipping.html) instead.

If you were looking for a marker for the commented out tests, I think `xfail` (expected failure) is more appropriate here than `skip`: it will force the test case to execute, but considers the result good if it fails rather than if if it succeeds.